src/utils/Utils.ts

```diff
@@ -475,11 +475,11 @@ export default class Utils {
   }
 
   public static computeSimplePrice(pricePerkWh: number, consumptionWh: number): number {
-    return Utils.truncTo(pricePerkWh * (consumptionWh / 1000), 6);
+    return Utils.roundTo(pricePerkWh * (consumptionWh / 1000), 6);
```

Comment:
You can't round here: rounding means you charge the user for not existent consumption and that's illegal.

Reply:
Please check the issue.

Reply:
Rounded pricing computation are still not legal in most countries around the world. 
The issue is with the truncTo() helper: it should properly return the truncated values after the comma.

Reply:
The fix in OCPPUtils and in SimplePricingIntegration classes above should be sufficient to solve this issue.
In this method I would keep all the decimals so we could remove the Utils call.

Reply:
I agree to keep all the decimals here. Can we have problems with the database?

Reply:
> I agree to keep all the decimals here. Can we have problems with the database?

I don't think so  :-)

> MongoDb added support for Decimal data type in 3.4 version. It is also available from the shell. 3.4 adds support for the decimal128 format with the new decimal data type. The decimal128 format supports numbers with up to 34 decimal digits (i.e. significant digits) and an exponent range of −6143 to +6144